### DISCUSSION TITLE
Formatting transcriptions for the usfm-bibles section

### DISCUSSION THREAD

**Discussion by jbj99** [openenglishbible/Open-English-Bible#358] [Q&A] [answered]
Hi Peter @prust,
Just moving this discussion to Q&A as it probably fits better here rather than in the general discussion forum. 
I have had a crack at transcribing a few psalms from Moffatt (https://github.com/jbj99/usfm-bibles/blob/master/Moffat/19-Psalms.usfm) and would appreciate any feedback on whether I am on the right track with the formatting. 
I couldn't find any 'key' after a quick look-around, but maybe I was looking in the wrong places. And again, I'm not really a coder so maybe it is meant to be obvious 🤦‍♂️
What I figured from your previous transcriptions is the following: 

\c means chapter number
\v means verse number
\p means new paragraph
\b means a line break in the text
\q indicates the indentation level. \q is the 'main' level (no indentation); \q2 is 1 indentation level; \q3 is 2 indentation levels, etc.

Is this correct? And does my formatting for these psalms match what you would do for these texts (https://archive.org/details/oldtestamentnewt0000unse/page/102/mode/2up)? 

Other than that, the only query I have at the moment is around the use of "\q". Does it have to be used in the line before every verse ("\v"), even if it is continuing at the same indentation level as the previous verse/line? 
There are also some verses which began at the q2 indentation level, so I have used "\q2" in the line before "\v n"... is that right? Your feedback is much appreciated. Hopefully I will be able to chip away at a few more psalms and help get this project completed. 

Kind regards,
Joel

**Comment by prust** [comment, marked as answer]
@jbj99:

> any feedback on whether I am on the right track with the formatting

Nice work! I proofread it with a fine-tooth comb and it looks exactly as I would have transcribed it except in two places:

* Psalm 102:4 has a short hyphen instead of a longer dash
* Psalm 102:23 needs a break (`\b`) before it

I just glanced at my recent transcription and immediately [saw a typo](https://github.com/openenglishbible/usfm-bibles/pull/3/commits/680766d7c73), so apparently I could use some proofing too :-)

> I couldn't find any 'key' after a quick look-around, but maybe I was looking in the wrong places

I'm glad you asked! My memory is fuzzy, since it's been a while since I started on this, but I think I just did what you did and followed the examples of previous transcriptions instead of finding documentation. Or if I did find documentation, I don't think I saw the documentation that is specifically on how to handle poetry. But thanks to your question, I searched and found it:

* Documentation for USFM syntax: https://ubsicap.github.io/usfm/
* Section specifically about Poetry: https://ubsicap.github.io/usfm/poetry/index.html

I did ask for a review of my transcription from @openenglishbible and he said:

> It looks reasonable to me. When you've done a book I can run it through Paratext and some tests to see if it is complying USFM .

So I think I've been doing a reasonable job, but looking over the Poetry documentation, I'm sure there are special poetry-specific cases that I haven't handled correctly. It would probably be wise to take the time to figure out how to install & use Paratext, to see if it detects any errors with what I've done on the Psalms so far (or if you're interested, that would be helpful, my time is pretty limited these days).

> the use of "\q". Does it have to be used in the line before every verse ("\v"), even if it is continuing at the same indentation level as the previous verse/line?

I believe so (though I'm certainly no expert). Looking at the USFM Poetry docs, it sounds like this distinguishes it as a poetic line, as opposed to the prose found in other books.

> There are also some verses which began at the q2 indentation level, so I have used "\q2" in the line before "\v n"... is that right?

Yes, I'm pretty sure this is the right way to do it, and it's the way I've done it.

**Reply by jbj99** [reply]
Thanks for your detailed response @prust, much appreciated! Those USFM links are also very helpful.
